test/e2e/adminapi.go

```diff
@@ -0,0 +1,66 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+)
+
+func adminRequest(ctx context.Context, method, path string, params url.Values, in, out interface{}) (*http.Response, error) {
+	if os.Getenv("RP_MODE") != "development" {
```

Comment:
Is this still needed if we check before each test suit?

Reply:
I would keep it to prevent people from using it in tests without the check.

At some point I think we will make it work in non-development env.

Reply:
I doubt this time will come very soon considering geneva API limitations.